config/internal/visualization-server/deployment.yaml.tmpl

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ds-pipeline-visualizationserver-{{.Name}}
+  namespace: {{.Namespace}}
+  annotations:
+    alpha.image.policy.openshift.io/resolve-names: 'visualization-server:1.3.1'
+    image.openshift.io/triggers: >-
+      [{"from":{"kind":"ImageStreamTag","name":"visualization-server:1.3.1"},"fieldPath":"spec.template.spec.containers[?(@.name==\"ds-pipeline-visualizationserver\")].image"}]
+  labels:
+    app: ds-pipeline-visualizationserver-{{.Name}}
+    component: data-science-pipelines
+spec:
+  selector:
+    matchLabels:
+      app: ds-pipeline-visualizationserver-{{.Name}}
+      component: data-science-pipelines
+  template:
+    metadata:
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+      labels:
+        app: ds-pipeline-visualizationserver-{{.Name}}
+        component: data-science-pipelines
+    spec:
+      initContainers:
+        - name: image-checker
+          image: 'quay.io/modh/odh-deployer:v1.15.0-10-rhods'
```

Comment:
This image reference doesn't seem right.

Reply:
Are there no upstream-y images we can use instead?

Reply:
removed with visualization server